contracts/cw1-whitelist/Cargo.toml

```diff
@@ -17,11 +17,9 @@ backtraces = ["cosmwasm-std/backtraces"]
 # use library feature to disable all instantiate/execute/query exports
 library = []
 test-utils = []
+boot = ["dep:boot-contract-derive","dep:boot-fns-derive", "dep:boot-core"]
```

Comment:
Probably better to re-export the macro's in `boot-core` re-exporting the macro's

Reply:
I'm not sure what you mean here

Reply:
Instead of importing the macro crates we export the macros in `lib.rs` of boot-core: 

```rust
pub use boot_contract_derive::contract;
pub use boot_fns_derive::{ExecuteFns, QueryFns};
```

So that they are available by the `boot-core` consumer.

Reply:
That's done now